Configurations/windows-makefile.tmpl

```diff
@@ -14,18 +14,18 @@
 
  (our $sover_dirname = $config{shlib_version_number}) =~ s|\.|_|g;
 
+ my $build_scheme = $target{build_scheme};
+ my $build_flavour = $build_scheme->[$#$build_scheme];
```

Comment:
In the context of "VC-W32" vs. "VC-WOW" it might be appropriate to name this variable as `$install_flavour`.

Reply:
Who says that the value is limited to installation stuff? The point was for the value to indicate a categorisation, to be used as we see fit.

Reply:
> Who says?

I do :-)

> categorisation

It still categorizes things :-) But on serious note, this is used [to pull environment variables that is then used] as pre-compiled path reflecting where thins will be installed, and of course as destination directory for nmake install. In both cases there is notion of "install" involved, so I'd say it's not inappropriate name...